labelbox/schema/organization.py

```diff
@@ -34,3 +42,155 @@ def __init__(self, *args, **kwargs):
     users = Relationship.ToMany("User", False)
     projects = Relationship.ToMany("Project", True)
     webhooks = Relationship.ToMany("Webhook", False)
+
+    def invites(self) -> PaginatedCollection:
+        """ List all current invitees
+        
+        Returns:
+            A PaginatedCollection of Invite objects
+
+        """
+        query_str = """query GetOrgInvitationsPyApi($from: ID, $first: PageSize) {
+                organization { id invites(from: $from, first: $first) { 
+                    nodes { id createdAt organizationRoleName inviteeEmail } nextCursor }}}"""
+        return PaginatedCollection(
+            self.client,
+            query_str, {}, ['organization', 'invites', 'nodes'],
+            Invite,
+            cursor_path=['organization', 'invites', 'nextCursor'],
+            experimental=True)
+
+    def _assign_user_role(self, email: str, role: Role,
+                          project_roles: List[ProjectRole]) -> Dict[str, Any]:
+        """
+        Creates or updates users. This function shouldn't directly be called. 
+        Use `Organization.invite_user`
+
+        Note: that there is a really unclear foreign key error if you use an unsupported role.
+        - This means that the `Roles` object is not getting the right ids
+        """
+        if self.client.get_user().email == email:
+            raise ValueError("Cannot update your own role")
+
+        data_param = "data"
+        query_str = """mutation createInvitesPyApi($%s: [CreateInviteInput!]){
+                    createInvites(data: $%s){  invite { id createdAt organizationRoleName inviteeEmail}}}""" % (
+            data_param, data_param)
+
+        projects = [{
+            "projectId": x.project.uid,
+            "projectRoleId": x.role.uid
+        } for x in project_roles]
+
+        res = self.client.execute(
+            query_str, {
+                data_param: [{
+                    "inviterId": self.client.get_user().uid,
+                    "inviteeEmail": email,
+                    "organizationId": self.uid,
+                    "organizationRoleId": role.uid,
+                    "projects": projects
+                }]
+            },
+            experimental=True)  # We prob want to return an invite
+        # Could support bulk ops in the future
+        invite_info = res['createInvites'][0]['invite']
+        return invite_info
+
+    def invite_user(self,
+                    email: str,
+                    role: Role,
+                    project_roles: List[ProjectRole] = []) -> Invite:
```

Comment:
default argument as array can become tricky because this array is a singleton instance for the whole module

`project_roles = Union[List, None]` might be better

Reply:
fixed. Changed it to `project_roles: Optional[List[ProjectRole]] = None`